src/Oro/Bundle/EntityBundle/Form/Type/EnumFormType.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace Oro\Bundle\EntityBundle\Form\Type;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Form\FormEvent;
+use Symfony\Component\Form\FormEvents;
+use Symfony\Component\Validator\ExecutionContextInterface;
+
+use Oro\Bundle\EntityExtendBundle\Entity\AbstractEnumValue;
+use Oro\Bundle\EntityExtendBundle\Tools\ExtendHelper;
+
+/**
+ * Used for validation and automatic identifier setter
+ */
+class EnumFormType extends AbstractType
```

Comment:
please rename to 'CustomEnumType' as it looks a bit inconsistent with definition in form_type.yml